Veil-light-wallet-Linux.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Installing the Veil Light Wallet on Linux (Ubuntu)</title>
+    <link href="ohcee_guides.css" rel="stylesheet" type="text/css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+</head>
+
+<body>
+    <h1>Installing the Veil Light Wallet on Linux (Ubuntu)</h1>
+    <h4>Follow this guide to install the Veil Light Wallet on your Linux machine running Ubuntu.</h4>
+
+    <ol>
+        <h3>For this guide, we'll be using Ubuntu 22.04.3 LTS.</h3>  
+    </ol>
+
+    <h2>Requirements:</h2>
+    <ol>
+        <li>A computer running Ubuntu.</li>
+        <li>A stable internet connection.</li>
+    </ol>
+
+    <h2>Downloading the Veil Light Wallet:</h2>
+    <ol>
+        <li>Visit the GitHub repository to obtain the latest version of the light wallet: <a href="https://github.com/steel97/veil_wallet/releases" target="_blank" rel="noopener noreferrer">Veil Light Wallet Releases</a>.</li>
+        <ul>For Linux, look for the appropriate release for Ubuntu.</ul>
+        <figure>
+            <img src="light-wallet-pictures/veil_wallet_linux.png" alt="veil_wallet_linux_photo">
+            <figcaption>Select the Linux version.</figcaption>
+        </figure>
+        <li>Download the package suitable for Ubuntu.</li>
+    </ol>
+
+    <h2>Installing the Veil Light Wallet:</h2>
+    <ol>
+        <li>Unzip the package to a location you know. For this example I'm saving it to my desktop for simplicity.</li>
+        <li>If the files are not in a folder, create a new folder and drag the files into it.</li>
+        <figure>
+            <img src="light-wallet-pictures/new_folder_w_files.png" alt="new_folder_w_files">
+            <figcaption>New folder on top.<br>
+            Downloaded wallet files on bottom.</figcaption>
+        </figure>
+        <li>Open the terminal and navigate to your wallet location. If you have the folder on your desktop, <br>
+            type <code>cd Desktop</code> and then <code>cd veil\ light\ wallet/</code> (or whatever you named the folder).</li>
+            <figure>
+                <img src="light-wallet-pictures/light_wallet_example2.png" alt="light_wallet_example">
+            </figure>
+        <li>Once in the folder, type <code>chmod +x veil-wallet</code> to make the veil-wallet file executable.</li>
```

Comment:
Would be good to add information about changing permissions without terminal (in ubuntu it's possible in file manager by right-clicking on executable `Properties > Allow executing as program`. I had this option in 23.10, but it also should be available in 22.04)

Reply:
Totally forgot about that way. Will add with pictures.